www/css/main.diary.css

```diff
@@ -36,9 +36,37 @@
   background:#7570b3!important;
   color:whitesmoke !important;
 }
-.angular-leaflet-map {
-  width: 300px;
-  height: 150px;
+
+
+/* leaflet */
+/* ----------- iPhone 5 and 5S ----------- */
```

Comment:
what about android phones?
this is not an iOS-only app

I know that leaflet doesn't support `ng-class` etc but you should be able to use the window-size to set the size, using javascript if necessary.

Here are three suggestions that I provided earlier.

According to this:
http://leafletjs.com/examples/mobile.html
% is supported at least for height.

Another workaround is to read the browser width and set the width based on that in javascript.
https://www.drupal.org/node/2086539

Or try applying this patch, since we have a patched leaflet anyway. You'll have to find where that code is in the concatenated leaflet.js that we include.
https://www.drupal.org/files/issues/map-height-relative-2086539-9.patch